liblll/CompilerState.cpp

```diff
@@ -55,10 +55,10 @@ void CompilerState::populateStandard()
 	"(def 'msg (to data) { [0]:data (msg allgas to 0 0 32) })"
 	"(def 'create (value code) { [0]:(msize) (create value @0 (lll code @0)) })"
 	"(def 'create (code) { [0]:(msize) (create 0 @0 (lll code @0)) })"
+	"(def 'sha3 (loc len) (keccak256 loc len))"
 	"(def 'sha3 (val) { [0]:val (sha3 0 32) })"
```

Comment:
No, no conflict. The macro expansion takes into account the number of arguments so the two sha3 macros are distinguished correctly.